xayn-ai/src/embedding/utils.rs

```diff
@@ -110,6 +110,11 @@ pub fn cosine_similarity(a: ArrayView1<f32>, b: ArrayView1<f32>) -> f32 {
     pairwise_cosine_similarity([a.view(), b.view()])[[0, 1]]
 }
 
+/// Computes the cosine distance of two vectors.
+pub fn cosine_distance(a: ArrayView1<f32>, b: ArrayView1<f32>) -> f32 {
```

Comment:
Can't we have this in the de instead of splitting the code?

Reply:
yes we can do this. the only downside is that we have to add `ndarray` dependency back to de

Reply:
I don't think is a problem